### PR TITLE
feat(admin-sdk): add typed telemetry helpers for standard events

### PR DESCRIPTION
**Pull request by bubleg**
## What?

Adds typed helper functions for standard telemetry events: `trackPageView()` and `trackLinkVisited()`. The generic `dispatch()` remains available for custom events.

## Why?

Follow-up to #1099. Without helpers, extensions would invent their own event and property names for common events, making Amplitude data impossible to aggregate across extensions (e.g. `page_viewed` vs `page_view` vs `page_visited`).

## How?

- `trackPageView({ sw_route_from_href, sw_route_from_name, sw_route_to_href, sw_route_to_name, sw_route_to_query? })` — matches the `page_viewed` event shape from the Admin
- `trackLinkVisited({ sw_link_href, sw_link_type: 'internal' | 'external' })` — matches the `link_visited` event shape from the Admin
- Both wrap `dispatch()` with a fixed event name and typed properties enforced by TypeScript

Login/logout helpers were considered and intentionally excluded — those are admin lifecycle events already tracked by the Admin itself.

## Testing?

- 6 new unit tests covering all helpers and edge cases
- All 175 tests pass
- Lint and type checks pass

## Screenshots (optional)

N/A

## Anything Else?

N/A

### CODE REVIEW

File: packages/admin-sdk/src/telemetry/index.ts [hunk 1]
```diff
@@ -16,6 +16,31 @@ export const dispatch = createSender('telemetryDispatch');
...
+ * Track a page view, matching the `page_viewed` event emitted by the Admin.
+ * Use this instead of `dispatch` to ensure consistent property names.
+ */
+export function trackPageView(properties: {
+  sw_route_from_href: string,
+  sw_route_from_name: string | null,
+  sw_route_to_href: string,
+  sw_route_to_name: string | null,
+  sw_route_to_query?: string,
+}): Promise<void> {
```
Comment (SebastianFranze) [inline packages/admin-sdk/src/telemetry/index.ts:35]: Should we still allow additional props? To force only the route params?


```suggestion
export function trackPageView(properties: {
  sw_route_from_href: string,
  sw_route_from_name: string | null,
  sw_route_to_href: string,
  sw_route_to_name: string | null,
  sw_route_to_query?: string,
  [key: string]: string | string[] | number | boolean //  -> our trackable type in platform
}): Promise<void> {
```

File: packages/admin-sdk/src/telemetry/index.ts [hunk 2]
```diff
@@ -26,6 +31,7 @@ export function trackPageView(properties: {
   sw_route_to_href: string,
   sw_route_to_name: string | null,
   sw_route_to_query?: string,
+  [key: string]: TrackableType | undefined,
```
Comment (SebastianFranze) [inline packages/admin-sdk/src/telemetry/index.ts:34]: The undefined is not necessary. If it is not there it is not there 😄

**Review by SebastianFranze — APPROVED** [review]

File: packages/admin-sdk/src/telemetry/index.ts (hunk 2 above)
Comment (bubleg) [reply packages/admin-sdk/src/telemetry/index.ts:34]: my ide begs to differ

Bildschirmfoto 2026-04-09 um 10 51 09
